sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5767,6 +5899,33 @@ bool _pi_queue::useCopyEngine(bool PreferCopyEngine) const {
          (!isInOrderQueue() || UseCopyEngineForInOrderQueue);
 }
 
+pi_result _pi_queue::waitOnAllImmCmdLists() {
```

Comment:
It will be good to add a comment here about what is currently done and possible improvements here.

Reply:
OK.